README.md

```diff
@@ -177,17 +215,27 @@ Necessary changes are temporarily made to the system which are lost with system
 
 ### Install - auto-cpufreq daemon
 
-Necessary changes are made to the system for auto-cpufreq CPU optimization to persist across reboots. Daemon is deployed and then started as a systemd service. Changes are made automatically and live stats are generated for monitoring purposes.
+Necessary changes are made to the system for auto-cpufreq CPU optimization to persist across reboots. The daemon is deployed and then started as a systemd service. Changes are made automatically and live stats are generated for monitoring purposes.
+
+Install the daemon using this command (after installing auto-cpufreq):
 
 `sudo auto-cpufreq --install`
 
-After daemon is installed, `auto-cpufreq` is available as a binary and is running in the background. Its stats can be viewed by running: `auto-cpufreq --stats`
+After the daemon is installed, `auto-cpufreq` is available as a binary and is running in the background. Its stats can be viewed by running: `auto-cpufreq --stats`
+
+Alternatively, using just systemd:
```

Comment:
Let's leave out lines 226 to 232, because `sudo auto-cpufreq --install` will enable and start newly created auto-cpufreq systemd service ...

Reply:
If you wish, as you wrote in section below for snap, you can write something similar here like "After daemon is installed, you can verify the status of systemd service by running `systemctl status auto-cpufreq`"